files/en-us/glossary/tcp/index.html

```diff
@@ -11,7 +11,7 @@
 ---
 <p id="Summary"><strong>TCP (Transmission Control Protocol)</strong> is an important network {{Glossary("protocol")}} that lets two hosts connect and exchange data streams.  TCP guarantees the delivery of data and packets in the same order as they were sent.  Vint Cerf and Bob Kahn, who were DARPA scientists at the time, designed TCP in the 1970s.</p>
 
-<p>TCP role is to ensure the packets are reliably delivered, error free.  TCP has concurrence control, which means the initial requests start small, increasing in size to the levels of bandwidth the computers, servers, and network can support.</p>
+<p>TCP's role is to ensure the packets are reliably delivered, error free.  TCP has concurrence control, which means the initial requests start small, increasing in size to the levels of bandwidth the computers, servers, and network can support.</p>
```

Comment:
```suggestion
<p>TCP's role is to ensure the packets are reliably delivered, error free. TCP has concurrence control, which means the initial requests start small, increasing in size to the levels of bandwidth the computers, servers, and network can support.</p>
```

I think, an extra space sneaked in here, too.